lib/sms.rb

```diff
@@ -0,0 +1,18 @@
+class Sms
+  def send_text
+    require 'twilio-ruby'
+
+    # put your own credentials here
+    account_sid = 'AC33011a40dc7a73068b3aecbc38deb302'
+    auth_token = '2b6b55ca0e757662eae67ee5df0fcac0'
+
+    # set up a client to talk to the Twilio REST API
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    @client.account.messages.create({
+    	:from => '+441757322034',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Tab detected.